### DISCUSSION TITLE
How do I stop daily update checks and reboots?

### DISCUSSION THREAD

**Discussion by WitsendMS** [elementary/appcenter#2149] [Q&A]
How do I stop daily update checks and reboots? Horus 7.1 on Early 2009 iMac. I'm finding the daily update process very tedious and , dare I say it, Windows like! 
I moved to Elementary OS from OS X as it had come to the end of it's useful life and wouldn't do system updates as it was "too old". I ran a short project of the available Linux distro options and Elementary OS won hands down. I like it a lot. I have carried out a search of previous entries but haven't found a solution.

**Comment by danirabbit** [comment]
You definitely should not be getting updates that require a reboot every single day. At this point you should only be receiving security updates for OS 7.1 and they are not released on a daily basis 

Which packages are you getting updates for? Have you added a 3rd party software repository?

**Reply by WitsendMS** [reply]
Thank you for your prompt response.
I've just received the second notification for today.
Screenshots attached.
![Screenshot from 2024-05-31 16 27 16](https://github.com/elementary/appcenter/assets/10983373/12604591-bb41-4812-8fd0-4791833a696e)
![Screenshot from 2024-05-31 16 26 17](https://github.com/elementary/appcenter/assets/10983373/49a4dd9f-1811-418c-84f8-427e89ec1f9e)
I do have other software installed.

**Reply by danirabbit** [reply]
Hm yeah I mean that's just a regular security update from Ubuntu. It looks like the last time that package was updated was in March, but it also says that particular update was released on May 2nd https://changelogs.ubuntu.com/changelogs/pool/main/v/vim/vim_8.2.3995-1ubuntu2.17/changelog

There's some good information about Ubuntu's release process for security updates here: https://askubuntu.com/questions/167457/is-it-required-to-update-ubuntu-every-day-or-week

Updates are generally a good thing, it means some issue in your operating system has been fixed. You can choose not to install an update if you feel you don't need it, but since security updates are important it's recommended to install them as they are released

**Reply by WitsendMS** [reply]
Thank you for your reply. I'm aware that Linux updates are generally a good thing but it's the required restart that annoys. I haven't had this on other distros. I'm hoping it will be different in Elementary OS 8 as I've grown attached to it as an OS.

**Reply by WitsendMS** [reply]
Every day further updates. Yesterdays was the first time in a while that didn't request a restart, however, today it's a different story:
![Screenshot from 2024-06-06 04 37 43](https://github.com/elementary/appcenter/assets/10983373/abc643f0-358f-4bac-9d5c-d1f3cca7d208)

**Comment by WitsendMS** [comment]
Updates and restart again today.
![Screenshot from 2024-06-07 05 09 39](https://github.com/elementary/appcenter/assets/10983373/eb57209c-a774-4d6d-8b00-f75d3fd48a93)
